.github/workflows/ci.yml

```diff
@@ -9,25 +9,26 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v1
+    - name: Setup PostgreSQL
+      uses: harmon758/postgresql-action@v1
+      with:
+        postgresql version: '12'
+        postgresql db: 'whiteclaw_test'
+        postgresql user: 'postgres'
+        postgresql password: 'postgres'
     - name: Setup Python
       uses: actions/setup-python@v2
       with:
         python-version: '3.8.3'
     - name: Install
       run: |
         pip install -r requirements.txt
-        mkdir data/
-        curl "https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/vaccinations.csv" > data/vaccinations.csv
```

Comment:
Do we not need this as part of CI anymore? Or is this more of a CD thing? Bit confused

Reply:
We changed how we did this a while ago to always download the OWID CSVs on each ETL run instead of relying on a local file - so we don't need this step in CI or CD atm